eagles-food-backend/Controllers/OrganizationsController.cs

```diff
@@ -10,7 +10,7 @@
 namespace eagles_food_backend.Controllers
 {
     [ApiController]
-    [Route("api/organization")]
+    [Route("api/organizations")]
```

Comment:
https://mirage-nitrogen-8b7.notion.site/Team-Eagles-tweaked-endpoint-documentation-d88e149d8a7e4cbba466c2795710cf03 specifies organization, and it's tweaked benrobo's...lets keep to that